Installer/bin/libAprilTags.sh

```diff
@@ -19,6 +19,49 @@ function getProcPID() {
     echo 0
 }
 
+echoerr() { echo "$@" 1>&2; } # bypass capture by var assigning because std::err
+
 if [[ $1 == "pid" ]]; then
     getProcPID $2
+elif [[ $1 == "camIDs" ]]; then
+    cd "${0%/*}"/camerascanner
+    # run the go file in the most efficent way possible
+    arch=$(uname -m)
+    if [[ -f "scanner_${arch}" ]]; then
+        ret=$("./scanner_${arch}")
```

Comment:
Since aarch64 binaries can be ran on x86_64 systems (for whatever reason), it might be worth ignoring the check for computer architecture and using a single architecture-agnostic aarch64 binary.